test/karma.conf.js

```diff
@@ -17,6 +17,7 @@ module.exports = function setKarmaConfig(config) {
     browserDisconnectTimeout: 120000, // default 2000
     browserDisconnectTolerance: 1, // default 0
     browserNoActivityTimeout: 6 * 60 * 1000, // default 10000
+    pollingTimeout: (80 / 5 * 4) / (1600 / (60 * 5)) * 1000,
```

Comment:
- 80: concurrency
- 5: number of tasks in Circle
- 4: number of browsers
- 1600/60*5: number of max average request/s

Reply:
Best document this with actual code:

```js
const circleCIConcurrency = 80;
const browserstackConcurrency = 4;
```
etc.

Though I don't know why the number of jobs in CircleCI matters. There could be between 1 and 80 parallel test_browser running.

Reply:
Agree with adding const variables. I have used 80 to account for the worse possible case, when dependabot opens 40 PRs and the 80 concurrency is used.

We might need to wait for next week dependency update to test the solution. In any case, I'm actually glad we have dependabot, at first, I felt it was overwhelming for the CI, but now, it seems to be a great stress test to make sure it can scale to more daily contributions.

Reply:
>  I have used 80 to account for the worse possible case, when dependabot opens 40 PRs and the 80 concurrency is used.

I was not asking about concurrency but  "5: number of tasks CircleCI (only one uses BrowserStack)
"

Reply:
I would call it an approximation 😁

Reply:
What are you approximating?

Reply:
A make the approximation that builds are mostly run when pushing commits, not when rebuilding inside the UI.